### PR TITLE
set secureboot_enable_disable as not supported for MiracleLinux-8.x

### PR DESCRIPTION
**Pull request by ZouYuhua**
test result:

For miraclelinux 8.8:
image

For miraclelinux 9:
image

### CODE REVIEW

File: linux/secureboot_enable_disable/secureboot_enable_disable.yml [hunk 1]
```diff
@@ -19,6 +19,15 @@
...
 
+        - name: "Skip testcase '{{ ansible_play_name }}' for {{ guest_os_ansible_distribution }} {{ guest_os_ansible_distribution_major_ver }}"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: "Skip testcase '{{ ansible_play_name }}', it is not supported by {{ guest_os_ansible_distribution }} {{ guest_os_ansible_distribution_major_ver }}"
+            skip_reason: "Not Supported"
+          when:
+            - guest_os_ansible_distribution == 'MIRACLE'
+            - guest_os_ansible_distribution_major_ver | int == 8
+
```
Comment (keirazhang) [inline linux/secureboot_enable_disable/secureboot_enable_disable.yml:30]: You can directly update `check_secureboot_support_status.yml`, which already defined not supported GOS list there

File: linux/secureboot_enable_disable/check_secureboot_support_status.yml [hunk 2]
```diff
@@ -18,12 +18,14 @@
   vars:
     skip_msg: >-
       Skip test case {{ ansible_play_name }} because secure boot is not supported
-      by {{ vm_guest_os_distribution }}.
+      by {{ vm_guest_os_distribution }} {{ guest_os_ansible_distribution_ver }}.
```
Comment (keirazhang) [inline linux/secureboot_enable_disable/check_secureboot_support_status.yml:21]: `vm_guest_os_distribution` has already include distribution version. Please do not  change this line.

**Comment by ZouYuhua** [comment]
@keirazhang 
passed after code change:

miracle linux 8.8:
image

miracle linux 9:
image

File: linux/secureboot_enable_disable/check_secureboot_support_status.yml (hunk 2 above)
Comment (ZouYuhua) [reply linux/secureboot_enable_disable/check_secureboot_support_status.yml]: Thanks. Done
No need to run package to test this minor changes.

**Review by keirazhang — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**